pkg/gadget-collection/gadgets/advise/seccomp/gadget.go

```diff
@@ -345,7 +347,14 @@ func (t *Trace) containerTerminated(trace *gadgetv1alpha1.Trace, event container
 
 	namespacedName := fmt.Sprintf("%s/%s", event.Container.Namespace, event.Container.Podname)
 
-	r, err := generateSeccompPolicy(t.client, trace, b, event.Container.Podname, event.Container.Name, namespacedName, event.Container.OwnerReference)
+	// This field was fetched when the container was created
+	ownerRefernce, err := event.Container.GetOwnerReference()
+	if err != nil && !errors.Is(err, rest.ErrNotInCluster) {
+		return
```

Comment:
Do we really want to not generate the profile because it failed to retrieve the owner reference information? I am not sure, but for sure, we must add a log here to know why the profile wasn't generated:

```suggestion
		log.Errorf("getting owner reference: %s", err)
		return
```

Reply:
Less add a warning and keep generating the profile.